### PR TITLE
Almas

### PR DESCRIPTION
**Pull request by almasfiza**
Pull request for branch 'almas'. Dashboard with formatted upload button, working dropdown, and table.

### CODE REVIEW

File: client/src/containers/Analytics/Analytics.js [hunk 1]
```diff
@@ -1,53 +1,72 @@
-import React from 'react';
+import React from "react";
```
Comment (abefong54) [inline client/src/containers/Analytics/Analytics.js]: You shouldn't have touched the Analytics page

File: client/src/hooks/useGetClassVideos.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// import React from 'react';
```
Comment (abefong54) [inline client/src/hooks/useGetClassVideos.js]: this whole file can be deleted - don't need to push it

**Review by abefong54 — CHANGES_REQUESTED** [review]

File: client/src/components/Upload/Upload.css [hunk 3]
```diff
@@ -6,4 +6,35 @@
 .file-name-div p{
```
Comment (abefong54) [inline client/src/components/Upload/Upload.css:6]: You shouldn't be pushing these changes

File: client/src/components/Upload/Upload.js [hunk 4]
```diff
@@ -3,12 +3,12 @@ import React, { Component } from "react";
 import { Button } from "react-bootstrap";
```
Comment (abefong54) [inline client/src/components/Upload/Upload.js:3]: You shouldn't be touching this file

File: client/src/containers/Header/Header.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ import MenuIcon from '@material-ui/icons/Menu';
```
Comment (abefong54) [inline client/src/containers/Header/Header.js:10]: why did you add a comma here?

**Review by abefong54 — CHANGES_REQUESTED** [review]

File: client/src/containers/Header/Header.js (hunk 5 above)
Comment (almasfiza) [reply client/src/containers/Header/Header.js:10]: my bad :( I had tried adding style changes there, but then i removed it and missed the comma.

File: client/src/components/Upload/Upload.js (hunk 4 above)
Comment (almasfiza) [reply client/src/components/Upload/Upload.js:3]: changed it back to as it was.
